src/opnsense/mvc/app/views/OPNsense/Trust/authorities.volt

```diff
@@ -0,0 +1,132 @@
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+
+#}
+
+<script type="text/javascript">
+    $(document).ready(function () {
+        grid = $("#grid-cas").UIBootgrid(
+            {
+                search: '/api/trust/authorities/search',
+                del: '/api/trust/authorities/del/',
+                get: '/api/trust/authorities/getExisting/',
+                set: '/api/trust/authorities/setExisting/',
+                options: {
+                    formatters: {
+                        "commands": function (column, row) {
+                            return "<button type=\"button\" class=\"btn btn-xs btn-default command-edit\" data-row-id=\"" + row.uuid + "\" title=\"{{ lang._('edit CA') }}\" alt=\"{{ lang._('edit CA') }}\"><span class=\"fa fa-pencil\"></span></button> " +
+                                "<button type=\"button\" class=\"btn btn-xs btn-default command-exp\" data-type=\"crt\" data-row-id=\"" + row.uuid + "\" title=\"{{ lang._('export CA cert') }}\" alt=\"{{ lang._('export CA cert') }}\"><span class=\"glyphicon glyphicon-download\"></span></button>" +
+                                "<button type=\"button\" class=\"btn btn-xs btn-default command-exp\" data-type=\"key\" data-row-id=\"" + row.uuid + "\" title=\"{{ lang._('export CA private key') }}\" alt=\"{{ lang._('export CA private key') }}\"><span class=\"glyphicon glyphicon-download\"></span></button>" +
+                                "<button type=\"button\" class=\"btn btn-xs btn-default command-delete\" data-row-id=\"" + row.uuid + "\" title=\"{{ lang._('delete ca') }}\" alt=\"{{ lang._('delete ca') }}\"><span class=\"fa fa-trash-o\"></span></button>";
+                        },
+                        "Distinguished": function (column, row) {
+                            return row[column.id] +
+                            "<table width=\"100%\" style=\"font-size: smaller\"> \
+                                <tr> \
+                                <td>&nbsp;</td> \
+                            <td width=\"20%\">{{ lang._('Valid From') }}:</td> \
+                            <td width=\"70%\">" + row["startdate"] + "</td> \
```

Comment:
it should not contain width - this should be a CSS property .
The ":" should be in the translation call.

Reply:
please escape